JoinColony/colonyCDapp
# Fix: Colony Sidebar UI adjustments

## Description

![colony sidebar adjustments](https://github.com/user-attachments/assets/134426f5-64fe-4a96-8637-103e64138fd6)

## Testing

1. Connect Dev Wallet 1
2. Visit the planex colony
3. Observe the Colony Sidebar and verify that Activity Feed is now called Activity
4. Click the Colony Switcher component and verify that it maintains its hover state when the colony switcher popover is present

Resolves #3092
